biotope/2018-06-05_biotopeVision.md

```diff
@@ -0,0 +1,20 @@
+# Biotope is a framework which enables you to develop scalabale, platform-agnostic frontend component ecosystems
+
+## It helps developers to:
+- easily develop ui-components following a unified standard
+- keep focus on developing components, the architecture and dev environment is provided by Biotope.
+- build platform-agnostic ui components which can be used in single page applications as well as in traditional server side rendered cms projects
```

Comment:
We could complement this bullet point with: 
- build your UI components only once and use them together with any js framework you like

Reply:
what do you all think about:
- build your UI components only once and use them together with vanilla ES6 or the js framework of your choice, in single page applications as well as in traditional server side rendered cms projects

Reply:
Sounds great :) 👍